lang/py/build.sh

```diff
@@ -17,13 +17,14 @@
 
 set -e
 
-usage() {
-  echo "Usage: $0 {clean|dist|interop-data-generate|interop-data-test|lint|test}"
-  exit 1
-}
+cd "$(dirname "$0")" # If being called from another folder, cd into the directory containing this script.
+
+# shellcheck disable=SC1091
+source ../../share/build-helper.sh "Python"
 
-clean() {
-  git clean -xdf '*.avpr' \
+function command_clean()
```

Comment:
Please remove the `function` keyword from everywhere else instead of adding it here. There is very little practical difference, except that we should default to fewer characters when there is no reasonable argument for more.